### ISSUE TITLE
Lack of experimental details

### ISSUE THREAD

**Issue by chenshen03** [yuezunli/celeb-deepfakeforensics#4]
Thank you for your work. How do you construct the training set and the test set? How many frames are selected for each video?

**Comment by chenshen03** [comment]
@danmohaha Could you share the detailed experiment settings? Thank you very much!
